packages/react/src/patterns/sub-patterns/Card/__stories__/Card.stories.js

```diff
@@ -1,39 +1,48 @@
 import './index.scss';
-import { select, text, withKnobs } from '@storybook/addon-knobs';
+import {
+  select,
+  text,
+  withKnobs,
+  boolean,
+  object,
+} from '@storybook/addon-knobs';
 import { ArrowRight20 } from '@carbon/icons-react';
 import { Card } from '../';
 import React from 'react';
 import readme from '../README.md';
 import { storiesOf } from '@storybook/react';
 
+const props = {
+  cardProps: () => ({
```

Comment:
I might would order these.

1. image
2. eyebrow
3. heading
4. copy
5. cta

Type, and inverse can go above or below.

Reply:
@photodow Will make note to address for that component.

Reply:
done